dependencies/pip/requirements.in

```diff
@@ -4,36 +4,35 @@
 # More up-to-date version of django-digest than PyPI seems to have.
 # Also, python-digest is an unlisted dependency thereof.
 -e git+https://github.com/dimagi/python-digest@5c94bb74516b977b60180ee832765c0695ff2b56#egg=python_digest
--e git+https://github.com/dimagi/django-digest@52ba7edeb326efd97d5670273bb6fa8b0539e501#egg=django_digest
+-e git+https://github.com/dimagi/django-digest@419f7306443f9a800b07d832b2cc147941062d59#egg=django_digest
 
 # spss
 https://bitbucket.org/fomcl/savreaderwriter/downloads/savReaderWriter-3.3.0.zip#egg=savreaderwriter
 
 # ssrf
--e git+https://github.com/kobotoolbox/ssrf-protect@755efe16694273ce66060a51e04f973dc034ca4e#egg=ssrf_protect
+-e git+https://github.com/kobotoolbox/ssrf-protect@9b97d3f0fd8f737a38dd7a6b64efeffc03ab3cdd#egg=ssrf_protect
 
 # jnm's django storages
 # Necessary to use this fork until the resolution of
 # https://github.com/jschneier/django-storages/issues/566
 -e git+https://github.com/jnm/django-storages@s3boto3_accurate_tell#egg=django_storages
 
 # Regular PyPI packages
-pytz
-Django>=2,<3
+Django>=2.2,<2.3
 django-environ
 django-templated-email
 dpath
 gdata-python3
 modilabs-python-utils
 Pillow
-psycopg2-binary
-pymongo
+psycopg2>=2.8,<2.9
```

Comment:
Indeed.
What about removing `psycopg2` from requirements.in and add it to `external_services` (By the way I would rename this to `prod` which seems to me more obvious`).
In `dev_requirements.in` we could use `psycopg2-binary`?
Not sure we will gain some speed though. Because `dev_requirements.txt` are synced at run time.

Reply:
Let's consider it off topic. I find it strange that we are syncing at run time to begin with. I'm happy to discuss.

> Let your runtime be runtime and your build time be build time: for whatsoever is more than these cometh of evil